tests/zenko_e2e/backbeat/tests/api/metrics.js

```diff
@@ -0,0 +1,200 @@
+                        assert(throughputSize > 0);
+                        next();
+                    });
+                }),
+        ], done);
+    });
+
+    it('should get metrics when specifying metric type', done => series([
+        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100), next),
+        next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
+            undefined, next),
+        next => makeGETRequest('/_/backbeat/api/metrics/crr/all/backlog',
+            (err, res) => {
+                assert.ifError(err);
+                assert.equal(res.statusCode, 200);
+                getResponseBody(res, (err, body) => {
+                    assert.ifError(err);
+                    const keys = Object.keys(body);
+                    assert.strictEqual(keys.length, 1);
+                    assert.strictEqual(keys[0], 'backlog');
+
+                    assert(body.backlog.description);
+                    assert(body.backlog.results);
+                    const resultKeys = Object.keys(body.backlog.results);
+                    assert(resultKeys.includes('count'));
+                    assert(resultKeys.includes('size'));
+                    next();
+                });
+            }),
+    ], done));
+
+    it('should get metrics when specifying valid site', done => series([
+        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100), next),
+        next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
+            undefined, next),
+        next => makeGETRequest(
+            `/_/backbeat/api/metrics/crr/${destLocation}`,
+            (err, res) => {
+                assert.ifError(err);
+                assert.equal(res.statusCode, 200);
+                getResponseBody(res, (err, body) => {
+                    assert.ifError(err);
+                    const metricTypes = ['backlog', 'completions', 'throughput',
+                        'failures'];
+                    const keys = Object.keys(body);
+                    keys.forEach(key => {
+                        assert(metricTypes.includes(key));
+                        assert(body[key].description);
+                        assert(body[key].results);
+                        const resultKeys = Object.keys(body[key].results);
+                        assert(resultKeys.includes('count'));
+                        assert(resultKeys.includes('size'));
+                    });
+
+                    next();
+                });
+            }),
+    ], done));
+
+    it('should get bucket-level throughput metrics', done => {
+        let versionId;
+        const destinationKey = `${srcBucket}/${key}`;
+
+        series([
+            next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(10000),
+                next),
+            next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
+                undefined, next),
+            next => awsUtils.getObject(destBucket, `${destinationKey}`,
```

Comment:
You can get the version ID from the callback of the initial `putObject` to save making a request to AWS here.